src/test/utils/options/options.test.ts

```diff
@@ -41,53 +41,16 @@ describe('Options', () => {
         options.removeDeclarationByName(declaration.name);
     });
 
-    it('Does not throw if default value is within range for number declaration', () => {
+    it('Does not throw if default value is out of range for number declaration', () => {
         const declaration: NumberDeclarationOption = {
             name: 'test-number-declaration',
             help: '',
             type: ParameterType.Number,
             minValue: 1,
             maxValue: 10,
-            defaultValue: 5
```

Comment:
Let's make the default value explicit so that the test is obvious and doesn't break in the (admittedly very unlikely) situation that the default value is changed.

Reply:
Sorry, I don't understand. What do you mean by "make it explicit"? Do you mean to create a separate variable for it and use that in the object definition?

Reply:
I meant specifying `defaultValue: 0` instead of letting it default.

Reply:
Oh, sure. Somehow my brain ignored that the line has a red background (meaning it is no longer there).

Reply:
Well, it turns out it is already specified just that way: `defaultValue: 0`. Don't know why it does not show up as such here.

Reply:
Doh! So it is, diff is great until it isn't. Looks good to me.